cli/core/pkg/config/defaults.go

```diff
@@ -22,7 +22,7 @@ var (
 	DefaultStandaloneDiscoveryImagePath  = ""
 	DefaultStandaloneDiscoveryImageTag   = ""
 	DefaultStandaloneDiscoveryName       = "default"
-	DefaultStandaloneDiscoveryNameLocal  = "default-local"
+	DefaultStandaloneDiscoveryNameLocal  = "default"
```

Comment:
nit: A comment explaining the reason for the rename could be useful.

Reply:
Added the doc